cmd/src/actions_exec_logger.go

```diff
@@ -141,6 +141,21 @@ func (a *actionLogger) RepoWriter(repoName string) (io.Writer, bool) {
 	return w, ok
 }
 
+func (a *actionLogger) InfoPipe(prefix string) io.Writer {
+	if !a.verbose {
+		return ioutil.Discard
+	}
+	stdoutPrefix := fmt.Sprintf("%s -> [STDOUT]: ", yellow.Sprint(prefix))
+	stdout := textio.NewPrefixWriter(os.Stderr, stdoutPrefix)
+	return io.Writer(stdout)
```

Comment:
It's called `stdout` but writes to `os.Stderr`. Why? Shouldn't this be independent of "STDOUT" or "STDERR"? If not, then I'd rename both methods to `StdoutPipe` and `StderrPipe`.